content/general/concepts/file-paths/file-paths.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'File Paths'
+Description: 'A file path describes the location of a file in a web sites folder structure. They are used to link to external files.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'File Paths'
+  - 'Files'
+  - 'URL'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A file path describes the location of a file in a programs folder structure. They are used to link files, like:
+
+- Websites
+- Images
+- Videos
+- Other programs
+
+## Absolute File Paths
+
+URL paths in HTML can be absolute paths, like a full URL, for example:
```

Comment:
A URL is not the same as a file path. A web server can change the URL to point to a completely different file path on the local machine.  I think this should be talking primarily about file paths on the local machine, then discuss the relationship to URLs and relative paths. It should focus as much or more on using them to navigate a computer's file system as it does using them in a web context.